apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -107,14 +106,14 @@ function WpcomBlockEditorNavSidebar() {
 
 		observer.current.disconnect();
 
-		const nonSparse = itemRefs.current.filter( Boolean ) as HTMLElement[];
+		/*		const nonSparse = itemRefs.current.filter( Boolean ) as HTMLElement[];
 
 		// We only need to observe the first and last item in the list, might be better for performance
```

Comment:
@p-jackson Could you comment on whether we need this item refs? Was it for loading/scrolling?

Thanks!

Reply:
Oh I see, to anchor the create new post to the bottom if the scrollbar doesn't exist...

Reply:
Oh yup, I think you figured it out. It's only needed for fancy styling when the button is stuck to the bottom. So not needed since the button's always at the bottom now.

It was a whole bunch of extra complexity just to have that line appear so happy to be rid of it.